contracts/interfaces/IRelayRecipient.sol

```diff
@@ -8,11 +8,11 @@ pragma solidity ^0.6.2;
 abstract contract IRelayRecipient {
 
     /**
-     * return the forwarder we trust to forward relayed transactions to us.
+     * return if the forwarder is trusted to forward relayed transactions to us.
      * the forwarder is required to verify the sender's signature, and verify
      * the call is not a replay.
      */
-    function getTrustedForwarder() public virtual view returns(address);
```

Comment:
I'd leave here the getter method, maybe just named `getDefaultForwarder()`

Reply:
I realized your point. I really did.

Reply:
yes, I think `getDefaultForwarder()` should be here, instead of introducing yet another single-function interface.